tests/integration/docker/swarm.bats

```diff
@@ -74,6 +97,39 @@ setup() {
 	done 
 }
 
+@test "check that replicas has two interfaces" {
+	REPLICAS_UP=(`$DOCKER_EXE ps -a | tail -n +2 | awk '{ print $1 }'`)
+	for i in ${REPLICAS_UP[@]}; do
+		# here we are checking that each replica has two interfaces 
+		# and they should be always eth0 and eth1
+		$DOCKER_EXE exec $i bash -c "ip route show | grep -E eth0 && ip route show | grep -E eth1"
+	done
+}
+
+@test "check service ip among the replicas" {
+	service_name=`$DOCKER_EXE service ls | tail -n +2 |  awk '{ print $1 }'`
+	ip_service=`$DOCKER_EXE service inspect $service_name | grep Addr | cut -d':' -f2 | cut -d'"' -f2 |  cut -d'/' -f1`
+	REPLICAS_UP=(`$DOCKER_EXE ps -a | tail -n +2 | awk '{ print $1 }'`)
+	for i in ${REPLICAS_UP[@]}; do
+		# here we are checking that all the 
+		# replicas have the service ip
+		$DOCKER_EXE exec $i bash -c "ip a | grep $ip_service"
+	done
+}
+
+@test "ping among replicas on their gateway ip" {
+	REPLICAS_UP=(`$DOCKER_EXE ps -a | tail -n +2 | awk '{ print $1 }'`)
+	IPS_REPLICAS=(`$DOCKER_EXE network inspect docker_gwbridge | grep IPv4Address | cut -d':' -f2 | cut -d'"' -f2 |  cut -d'/' -f1 | head -n -1`)
```

Comment:
This gives the gateway network IPs. You still need to post process them
$DOCKER_EXE network inspect docker_gwbridge --format='{{range .Containers}} {{if ne .Name "gateway_ingress-sbox"}}{{.IPv4Address}}{{end}}{{end}}'